src/handleIncomingRequest.ts

```diff
@@ -27,9 +39,11 @@ export const createIncomingRequestHandler = (
     }
 
     // Retrieve request URL parameters based on the provided mask
-    const params = relevantRequestHandler.mask
-      ? match(relevantRequestHandler.mask, req.url).params
-      : {}
+    const params =
+      (relevantRequestHandler.mask &&
+        match(resolveRequestMask(relevantRequestHandler.mask), req.url)
```

Comment:
With the latest release of `node-match-path` it returns explicit `null` in case there are no parameters. Handling that by introducing this change.